packages/svelte2tsx/src/emitDts.ts

```diff
@@ -5,6 +5,8 @@ import { svelte2tsx } from './svelte2tsx';
 export interface EmitDtsConfig {
     declarationDir: string;
     svelteShimsPath: string;
+    svelteJsxPath?: string;
+    svelteNativeJsxPath?: string;
```

Comment:
Another option is adding a `[string`] option for additional paths instead of just listing these out individually, and another alternative is `emitDts` just finding & reading these itself (I wasn't sure why it's structured this way right now). *Another* option is combining all of the declarations into one file. 

Obviously you folks would know best, I just opened this PR to get the ball rolling.